dist/Dockerfile

```diff
@@ -4,12 +4,23 @@ FROM openjdk:8
 
 RUN apt-get update -y && apt-get install -y iptables stress
 
+ENV HOME=/opt/atomix
+ENV ATOMIX_DIR=/var/lib/atomix
+ENV DATA_DIR=${ATOMIX_DIR}/data
+
 ARG VERSION
-RUN mkdir -p /opt/atomix
-COPY target/atomix.tar.gz /opt/atomix/atomix.tar.gz
-RUN tar -xvf /opt/atomix/atomix.tar.gz -C /opt/atomix && rm /opt/atomix/atomix.tar.gz
 
-WORKDIR /opt/atomix
+RUN groupadd -g 2000 atomix && \
```

Comment:
might be safer avoid hard coded group and user ids.

Reply:
if we don't hard code it where do we have this value from?

Reply:
Use names instead?